<launch>

    <node pkg="spatial" type="parse_camera_time" name="parse_camera_time" output="screen">
        <param name="bag_path" type="string" value="/media/sam/T7/20210424/2021-04-24-17-09-02.bag"/>
        <param name="write_path" type="string" value="$(find spatial)/data/inc_time"/>
        <param name="is_img_topic" type="bool" value="true"/>
        <param name="topic_name" type="string" value="/left_camera/image"/>
    </node>

</launch>